pkg/cmd/admin/policy/modify_roles.go

```diff
@@ -71,6 +72,7 @@ func NewCmdAddRoleToGroup(name, fullName string, f *clientcmd.Factory, out io.Wr
 		},
 	}
 
+	cmd.Flags().StringVarP(&options.RoleBindingName, "rolebinding-name", "N", "", "name of the rolebinding to modify or create")
```

Comment:
`name of the rolebinding to modify or create. If left empty, overrides the first rolebinding found for the given role`